content/save-load-arrays.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<p><em>First, we have to create a single 2D array from our x2 1D arrays.</em></p><p> </p><p>I think the <code>x2</code>  here is a typo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Not a typo, but also not clear on a third read. </p><p>It was meant to be shorthand for times 2 1D arrays. Maybe there is a clearer way to say "two 1D arrays". So far @bjnath has been my new best friend for clear and concise writing. </p>

Reply:
@cooperrc
> my new best friend for clear and concise writing.

Thank you!